src/generator-runner.js

```diff
@@ -114,6 +114,15 @@ export default function (plopfileApi, flags) {
 		// data can also be a function that returns a data object
 		if (typeof cfgData === 'function') { cfgData = await cfgData(); }
 
+		// check if action should run
+		if (typeof cfg.when === 'function') {
+			const stubTrue = () => true;
+			const reasonToSkip = await (cfg.when || stubTrue)(data);
```

Comment:
When would `cfg.when` be falsy here? It seems that it's impossible to both have `typeof === 'function'` and it be falsey. Maybe I'm missing something?

Reply:
Good catch. That was left over from before it was in an if statement.